src/librustc_llvm/ffi.rs

```diff
@@ -505,17 +505,6 @@ pub mod debuginfo {
 
 pub enum ModuleBuffer {}
 
-// Link to our native llvm bindings (things that we need to use the C++ api
-// for) and because llvm is written in C++ we need to link against libstdc++
-//
-// You'll probably notice that there is an omission of all LLVM libraries
-// from this location. This is because the set of LLVM libraries that we
-// link to is mostly defined by LLVM, and the `llvm-config` tool is used to
-// figure out the exact set of libraries. To do this, the build system
-// generates an llvmdeps.rs file next to this one which will be
-// automatically updated whenever LLVM is updated to include an up-to-date
-// set of the libraries we need to link to LLVM for.
```

Comment:
Isn't this comment still largely applicable with some editing?

Reply:
Sort of, but there's no longer anything here to anchor it to, and the relevant code (in build.rs) is rather self explanatory.

Reply:
Ok yeah on rereading let's leave out